lib/fluent/plugin/exception_detector.rb

```diff
@@ -223,14 +223,22 @@ def push(time_sec, record)
       force_flush if @max_lines > 0 && @messages.length == @max_lines
     end
 
+    def combined_message
+      combined = ''
+      @messages.each do |m|
+        combined << $RS if !combined.empty? && !combined.end_with?("\n")
```

Comment:
This is a static value that should never change. How about making it a class variable instead?